src/reducers/picklist/__tests__/PicklistReducer-test.js

```diff
@@ -482,4 +483,71 @@ describe('PicklistReducer', () => {
     }
     assert.deepStrictEqual(newState, expectedState, 'Should not mark any item')
   })
+
+  it('Refresh node updates path', () => {
+    const newState = picklistReducer(testState, {
+      type: LOAD_ONE_NODE_SUCCESS,
+      result:{
+        path: ',1,',
+        pathNames: [{ id: 1}] },
+      id: 1
+    })
+    const expectedState =  {
+      NODE: [
+        {
+          marked: false,
+          path: [],
+          value: {
+            ...node1
+          }
+        },
+        {
+          marked: false,
+          path: [],
+          value: node2
+        },
+        {
+          marked: false,
+          path: [],
+          value: node3
+        }
+      ],
+      OBJECT: DEFAULT_OBJECTS
+    }
+    assert.deepStrictEqual(JSON.stringify(expectedState), JSON.stringify(newState), 'Should update path for node 1')
+  })
+
+it('Refresh node updates path works', () => {
```

Comment:
this test is either not doing what it should, or anything it all. I am not able to see what this test is testing.